install.sh

```diff
@@ -26,7 +41,7 @@ if [ -z "$1" ]; then
 fi
 
 if [ ! -e ngrok.service ]; then
-    git clone --depth=1 https://github.com/vincenthsu/systemd-ngrok.git
+    git clone --depth=1 https://github.com/elzdave/systemd-ngrok.git
```

Comment:
Thanks for your contribution. Could you change the git URL to this repository?

Reply:
@vincenthsu Yes, fixed it.